scripts/protocgen.sh

```diff
@@ -7,5 +7,5 @@ for dir in $proto_dirs; do
   protoc \
   -I. \
   --gogo_out=Mgoogle/protobuf/timestamp.proto=github.com/golang/protobuf/ptypes/timestamp,Mgoogle/protobuf/duration.proto=github.com/golang/protobuf/ptypes/duration,plugins=grpc,paths=source_relative:. \
-  $(find "${dir}" -name '*.proto')
+  "$(find "${dir}" -name '*.proto')"
```

Comment:
Does this work? If I'm reading this right, it will output all files as a single argument e.g. `"a b c"` not `"a" "b" "c"`. I suppose the intention is to handle file names containing whitespace, but I think that needs to be done via e.g. `find -exec` or something, since I don't think Bash supports it natively.

Reply:
```suggestion
  $(find "${dir}" -name '*.proto')
```